cli/node/service.py

```diff
@@ -52,3 +52,17 @@ def get_peers(self):
         return {
             'peers': peers,
         }, None
+
+    def get_info(self):
+        """
+        Get node information.
+        """
+        try:
+            information = loop.run_until_complete(self.service.node_management.get_node_info())
+
```

Comment:
Add `except RpcGenericServerDefinedError`.

Reply:
How to get this exception you know?

Reply:
This exception trow server, when server rpc_api have bugs or something going wrong on the server, for example, information not found, but all data was valid.

Reply:
How can I test it?

Reply:
As I already said, its error trow server ([list of errors that can trow server](https://docs.remme.io/remme-core/docs/apis/rpc.html#json-rpc-error-codes)).
If we validate all data on the client side we cannot test it, because of its many unexpected errors (for example bugs).
This will be our umbrella of unexpected errors on the server, that can not be tested.

Reply:
I understand you, but if I can't verify this I can't add this exception.